test-frame-log-collector/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,28 @@
+name = TFConfig
```

Comment:
seems same name like test-frame-common logger config, is it okey?

Reply:
Yeah I guess it's okay
I didn't have problem with that